src/App.js

```diff
@@ -11,10 +11,30 @@ function App() {
     dispatch(fetchStockHistory());
```

Comment:
- [ ] [REQUIRED] Good job so far making some changes required by the previous reviewer. But I can find a list of categories filter on your home page :cry:  Please make sure that the Home page has a list of categories that could be filtered using given parameters, e.g. category name to follow the project requirements also as a developer it is very important to pay attention to details :+1: 

#### Check the example below :arrow_down: 

![Screenshot from 2022-09-13 21-28-54](https://user-images.githubusercontent.com/6615394/190013844-d18cfe1f-b8ff-4903-aa01-59d4b2ebc22f.png)

- [ ] [OPTIONAL] Please make sure that all your tests passed, while you are updating your project please update your test also  because a successful test it s part of the project requirements :+1:

Reply:
Well resolved 👍🏽